src/app/common/ui-patternfly/syndesis-form-control.component.html

```diff
@@ -78,13 +78,13 @@
                                            [group]="control.at(idx)"
                                            [hasErrorMessaging]="item.hasErrorMessages"
                                            [model]="item"
-                                           [nestedTemplates]="templates"
+                                           [templates]="templates"
                                            [ngClass]="[item.cls.element.host, item.cls.grid.host]"
                                            (blur)="onFocusChange($event)"
                                            (change)="onValueChange($event)"
                                            (focus)="onFocusChange($event)"></syndesis-form-control>
```

Comment:
The problem with breaking custom element declarations into several lines per attribute like this is that code indenting becomes arbitrary and tightly coupled to the custom selector name, which is prone to change. Eg: Let's figure out we change the element tag prefix from `<syndesis-` to `<syn-` for brevity sake... Boom! All the indenting will look weird then and the template will become harder to read.

I'd suggest to stick to the default 2-spaces indenting for anything that has to be indented at some point, no matter what.

Reply:
That's fine, but let's do that in a separate PR.

Reply:
and definitely +1 2 space indenting, I just want to minimize the changes in this PR to this file to this fix, we can have a separate PR for reformatting the template files, I'd like to ensure we all have consistent formatting in our IDEs.

Reply:
yeah, actually this is the only file that has 4 spaces for some reason, was contributed but not sure why vscode refuses to reformat.  Sorted that via `yarn prettier`...

Reply:
yikes, `prettier` doesn't handle case-sensitive attributes, reverted that :-)